satellite/satellitedb/testdata/postgres.v32.sql

```diff
@@ -0,0 +1,317 @@
+
+INSERT INTO "accounting_timestamps" VALUES ('LastAtRestTally', '0001-01-01 00:00:00+00');
+INSERT INTO "accounting_timestamps" VALUES ('LastRollup', '0001-01-01 00:00:00+00');
+INSERT INTO "accounting_timestamps" VALUES ('LastBandwidthTally', '0001-01-01 00:00:00+00');
+
+INSERT INTO "nodes"("id", "address", "last_ip", "protocol", "type", "email", "wallet", "free_bandwidth", "free_disk", "major", "minor", "patch", "hash", "timestamp", "release","latency_90", "audit_success_count", "total_audit_count", "audit_success_ratio", "uptime_success_count", "total_uptime_count", "uptime_ratio", "created_at", "updated_at", "last_contact_success", "last_contact_failure", "contained", "disqualified", "reputation_audit_alpha", "reputation_audit_beta", "reputation_uptime_alpha", "reputation_uptime_beta") VALUES (E'\\153\\313\\233\\074\\327\\177\\136\\070\\346\\001', '127.0.0.1:55516', '', 0, 4, '', '', -1, -1, 0, 1, 0, '', 'epoch', false, 0, 0, 5, 0, 0, 5, 0, '2019-02-14 08:07:31.028103+00', '2019-02-14 08:07:31.108963+00', 'epoch', 'epoch', false, false, 1, 0, 1, 0);
+INSERT INTO "nodes"("id", "address", "last_ip", "protocol", "type", "email", "wallet", "free_bandwidth", "free_disk", "major", "minor", "patch", "hash", "timestamp", "release","latency_90", "audit_success_count", "total_audit_count", "audit_success_ratio", "uptime_success_count", "total_uptime_count", "uptime_ratio", "created_at", "updated_at", "last_contact_success", "last_contact_failure", "contained", "disqualified", "reputation_audit_alpha", "reputation_audit_beta", "reputation_uptime_alpha", "reputation_uptime_beta") VALUES (E'\\006\\223\\250R\\221\\005\\365\\377v>0\\266\\365\\216\\255?\\347\\244\\371?2\\264\\262\\230\\007<\\001\\262\\263\\237\\247n', '127.0.0.1:55518', '', 0, 4, '', '', -1, -1, 0, 1, 0, '', 'epoch', false, 0, 0, 0, 1, 3, 3, 1, '2019-02-14 08:07:31.028103+00', '2019-02-14 08:07:31.108963+00', 'epoch', 'epoch', false, false, 1, 0, 1, 0);
+INSERT INTO "nodes"("id", "address", "last_ip", "protocol", "type", "email", "wallet", "free_bandwidth", "free_disk", "major", "minor", "patch", "hash", "timestamp", "release","latency_90", "audit_success_count", "total_audit_count", "audit_success_ratio", "uptime_success_count", "total_uptime_count", "uptime_ratio", "created_at", "updated_at", "last_contact_success", "last_contact_failure", "contained", "disqualified", "reputation_audit_alpha", "reputation_audit_beta", "reputation_uptime_alpha", "reputation_uptime_beta") VALUES (E'\\363\\342\\363\\371>+F\\256\\263\\300\\273|\\342N\\347\\014', '127.0.0.1:55517', '', 0, 4, '', '', -1, -1, 0, 1, 0, '', 'epoch', false, 0, 0, 0, 1, 0, 0, 1, '2019-02-14 08:07:31.028103+00', '2019-02-14 08:07:31.108963+00', 'epoch', 'epoch', false, false, 1, 0, 1, 0);
```

Comment:
If you remove the `"reputation_audit_alpha", "reputation_audit_beta", "reputation_uptime_alpha", "reputation_uptime_beta"` columns from this `INSERT` and also the values `1, 0, 1, 0` this insert should allow to verify the column default values

Reply:
this is the v32 of the script.  it should require those values to be set, so as to verify them.  the v31 version of the script is essentially what you're asking for...the baseline against which we see if the default values get set.

Reply:
I don't get your answer.
I thought that the test applies the migration and after it compares the data of the database with the data inserted by this one, hence if you don't insert the values here, the defaults should apply as they are applied by the migration.

Reply:
v31 is run, the migrate scripts are applied, and the result is compared to v32.  thus v31 should not have defaults hard-coded, the migrate script should append default values and columns, and v32 should have everything hard-coded.

Reply:
> thus v31 should not have defaults hard-coded

This one was obvious to me that it has not to be touched.

> the migrate scripts are applied

That implies that the existing values of the database will get the defaults

> and the result is compared to v32.

For that reason I was saying that if the table contained the defaults, having an insert without the columns will check those defaults.

Howe due to the comment https://github.com/storj/storj/pull/2215#discussion_r294725233 it cannot be in that way because they don't have to be set on them.